### PR TITLE
Sb showcase

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
https://github.com/gravitee-io/issues/issues/6158

**Description**

Showcase for color, some angular component and typography used in apim

**Screenshots**
see link below

**Additional context**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-vhgcrrmltc.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/sb-showcase/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-lxohwxswor.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-peovitzbgm.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/sb-showcase/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: src/scss/gio-palettes.scss [hunk 1]
```diff
@@ -34,23 +34,6 @@ $gio-cyan-palette: mat.define-palette(
   dark
 );
 
-// Info color
-$gio-info-palette: mat.define-palette(
```
Comment (gaetanmaisse) [inline src/scss/gio-palettes.scss:38]: Ca c'est oui

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**